ember.md

```diff
@@ -135,11 +135,17 @@ actions: {
   }
 }
 
+//my-cheesy-component.hbs
+<form onSubmit=(action 'quesoChanger')>
+  <input type='text' value='gouda'/>
+  <button type='submit'>Change Cheese</button>
+</form>
+
 //parent component.js
 actions: {
-	newCheesePlease(cheeseType){
-		set(parentData, 'cheese', cheeseType);
-	}
+  newCheesePlease(cheeseType){
+    this.set('model.cheese', cheeseType);
```

Comment:
Why was the proper use of Ember.set replaced when the formatting changed?
Wouldn't it be `set(this, 'cheese', cheeseType);`?

Reply:
will update - was a quick chnage from twiddle